nasa-gibs/worldview
# WV-3186: Add SeaWiFS Corrected Reflectance layer

## Description

Fixes #WV-3186.

Added OrbView-2/SeaWiFS Corrected Reflectance (True Color) layer.

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-307.2217934331701,-117.66091291783245,206.82458059288138,114.26666248769827&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,SEAWIFS_ORBVIEW-2_GAC_True_Color,VIIRS_NOAA21_CorrectedReflectance_TrueColor(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2009-07-29-T17%3A29%3A04Z)
3. Verify expected result - verify the imagery looks good for the dates that it's available, layer description makes sense, image snapshot works. 


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
